beacon_node/beacon_chain/src/attestation_rewards.rs

```diff
@@ -50,9 +50,10 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
 
         match state {
             BeaconState::Base(_) => self.compute_attestation_rewards_base(state, validators),
-            BeaconState::Altair(_) | BeaconState::Merge(_) | BeaconState::Capella(_) => {
-                self.compute_attestation_rewards_altair(state, validators)
-            }
+            BeaconState::Altair(_)
+            | BeaconState::Merge(_)
+            | BeaconState::Capella(_)
+            | BeaconState::Deneb(_) => self.compute_attestation_rewards_altair(state, validators),
```

Comment:
My old comment when I was reviewing the PR, but be great to double-check 🙏 

> I think EIP-7045 doesn't require a change in the Altair attestation rewards calculation logic in the rewards API, because it uses `ParticipationCache` which reads the participation flags from the beacon state (which had the updated logic applied in deneb `per_block_processing`)